public/javascripts/handleAuth.js

```diff
@@ -0,0 +1,35 @@
+
+// Function check session to see if user is logged in. If not redirect to login page
+exports.notAuth = (req, res, next) => {
+    if(!req.session.isAuth) {
```

Comment:
I'd do a check for existence on the `user` object in the `req.session` i.e. `if (req.session.user)`